client/export.go

```diff
@@ -36,10 +37,23 @@ func (c *Client) Export(key, singleKeyExportFormat string) (string, error) {
 		sortedKeys = append(sortedKeys, secret.Key)
 		keySecretMap[secret.Key] = secret
 	}
+
+	var AppendMap map[string]string
+	AppendMap = make(map[string]string)
+	for _, val := range c.Append {
```

Comment:
What if one would like to add `--append k1:v1 --append k1:v2` or `--append k1\:v1`? I think you need more sanitization here.